src/store/plugins/ui/names.js

```diff
@@ -13,10 +13,20 @@ export default (store) => {
     namespaced: true,
     state: {
       names: {},
+      defaultNames: store.state.names ? store.state.names.defaultNames : {},
```

Comment:
we may use a function from lodash here:
```suggestion
      defaultNames: get(store.state, 'names.defaultNames', {}),
```